[2021-12-21 22:29:20,361] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: data_pipeline.super_resolution scheduled__2021-12-19T00:06:00+00:00 [queued]>
[2021-12-21 22:29:20,366] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: data_pipeline.super_resolution scheduled__2021-12-19T00:06:00+00:00 [queued]>
[2021-12-21 22:29:20,366] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-21 22:29:20,366] {taskinstance.py:1242} INFO - Starting attempt 1 of 4
[2021-12-21 22:29:20,366] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-21 22:29:20,374] {taskinstance.py:1262} INFO - Executing <Task(BashOperator): super_resolution> on 2021-12-19 00:06:00+00:00
[2021-12-21 22:29:20,376] {standard_task_runner.py:52} INFO - Started process 97389 to run task
[2021-12-21 22:29:20,378] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'data_pipeline', 'super_resolution', 'scheduled__2021-12-19T00:06:00+00:00', '--job-id', '88', '--raw', '--subdir', '/home/myamya/project/dockers/airflow/dags/hello_world.py', '--cfg-path', '/tmp/tmpedciaekv', '--error-file', '/tmp/tmpwhx_fhja']
[2021-12-21 22:29:20,378] {standard_task_runner.py:77} INFO - Job 88: Subtask super_resolution
[2021-12-21 22:29:20,396] {logging_mixin.py:109} INFO - Running <TaskInstance: data_pipeline.super_resolution scheduled__2021-12-19T00:06:00+00:00 [running]> on host myamya
[2021-12-21 22:29:20,418] {taskinstance.py:1412} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=yangjae
AIRFLOW_CTX_DAG_ID=data_pipeline
AIRFLOW_CTX_TASK_ID=super_resolution
AIRFLOW_CTX_EXECUTION_DATE=2021-12-19T00:06:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-19T00:06:00+00:00
[2021-12-21 22:29:20,418] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2021-12-21 22:29:20,419] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'docker run -it -v /home/myamya/project/image_files/:/home/myamya/project/image_files/ sure:0.3 /bin/bash /home/myamya/project/init.sh ']
[2021-12-21 22:29:20,423] {subprocess.py:85} INFO - Output:
[2021-12-21 22:29:22,088] {subprocess.py:89} INFO - /opt/conda/lib/python3.8/site-packages/torch/cuda/__init__.py:52: UserWarning: CUDA initialization: Found no NVIDIA driver on your system. Please check that you have an NVIDIA GPU and installed a driver from http://www.nvidia.com/Download/index.aspx (Triggered internally at  /opt/conda/conda-bld/pytorch_1603729096996/work/c10/cuda/CUDAFunctions.cpp:100.)
[2021-12-21 22:29:22,089] {subprocess.py:89} INFO -   return torch._C._cuda_getDeviceCount() > 0
[2021-12-21 22:29:23,105] {subprocess.py:89} INFO - Downloading: "https://github.com/xinntao/facexlib/releases/download/v0.1.0/detection_Resnet50_Final.pth" to /opt/conda/lib/python3.8/site-packages/facexlib/weights/detection_Resnet50_Final.pth
[2021-12-21 22:29:23,105] {subprocess.py:89} INFO - 
[2021-12-21 22:29:26,371] {subprocess.py:89} INFO -   0% 0.00/104M [00:00<?, ?B/s]  0% 248k/104M [00:00<00:48, 2.27MB/s]  2% 1.92M/104M [00:00<00:35, 3.07MB/s]  3% 3.25M/104M [00:00<00:26, 3.94MB/s]  7% 7.41M/104M [00:00<00:18, 5.41MB/s] 12% 12.0M/104M [00:00<00:13, 7.38MB/s] 15% 15.7M/104M [00:00<00:09, 9.74MB/s] 19% 19.6M/104M [00:00<00:07, 12.6MB/s] 22% 23.5M/104M [00:00<00:05, 15.9MB/s] 26% 27.1M/104M [00:00<00:04, 19.3MB/s] 29% 30.6M/104M [00:01<00:03, 22.2MB/s] 33% 34.2M/104M [00:01<00:02, 25.2MB/s] 36% 37.8M/104M [00:01<00:02, 28.0MB/s] 40% 41.4M/104M [00:01<00:02, 30.5MB/s] 43% 45.2M/104M [00:01<00:01, 32.8MB/s] 47% 49.3M/104M [00:01<00:01, 35.3MB/s] 51% 53.7M/104M [00:01<00:01, 38.0MB/s] 55% 57.8M/104M [00:01<00:01, 39.3MB/s] 60% 62.6M/104M [00:01<00:01, 41.9MB/s] 64% 67.1M/104M [00:01<00:00, 43.3MB/s] 68% 71.5M/104M [00:02<00:00, 44.2MB/s] 73% 76.0M/104M [00:02<00:00, 45.0MB/s] 77% 80.5M/104M [00:02<00:00, 45.7MB/s] 81% 85.0M/104M [00:02<00:00, 46.2MB/s] 86% 89.6M/104M [00:02<00:00, 46.7MB/s] 90% 94.2M/104M [00:02<00:00, 47.0MB/s] 95% 98.8M/104M [00:02<00:00, 47.4MB/s] 99% 103M/104M [00:02<00:00, 47.3MB/s] 100% 104M/104M [00:02<00:00, 39.5MB/s]
[2021-12-21 22:29:26,519] {subprocess.py:89} INFO - Downloading: "https://github.com/TencentARC/GFPGAN/releases/download/v0.2.0/GFPGANCleanv1-NoCE-C2.pth" to /opt/conda/lib/python3.8/site-packages/gfpgan/weights/GFPGANCleanv1-NoCE-C2.pth
[2021-12-21 22:29:26,519] {subprocess.py:89} INFO - 
[2021-12-21 22:29:34,260] {subprocess.py:89} INFO -   0% 0.00/332M [00:00<?, ?B/s]  0% 232k/332M [00:00<02:47, 2.07MB/s]  0% 1.65M/332M [00:00<02:04, 2.80MB/s]  1% 4.02M/332M [00:00<01:30, 3.81MB/s]  3% 8.52M/332M [00:00<01:05, 5.21MB/s]  4% 14.1M/332M [00:00<00:46, 7.17MB/s]  6% 18.3M/332M [00:00<00:34, 9.56MB/s]  7% 22.8M/332M [00:00<00:25, 12.6MB/s]  8% 27.0M/332M [00:00<00:20, 16.0MB/s]  9% 31.2M/332M [00:00<00:15, 19.8MB/s] 11% 35.6M/332M [00:01<00:13, 23.8MB/s] 12% 39.7M/332M [00:01<00:11, 27.5MB/s] 13% 44.0M/332M [00:01<00:09, 31.1MB/s] 14% 48.1M/332M [00:01<00:08, 33.7MB/s] 16% 52.2M/332M [00:01<00:08, 35.3MB/s] 17% 56.4M/332M [00:01<00:07, 37.5MB/s] 18% 60.5M/332M [00:01<00:07, 38.7MB/s] 19% 64.5M/332M [00:01<00:07, 39.7MB/s] 21% 68.5M/332M [00:01<00:06, 40.2MB/s] 22% 72.5M/332M [00:01<00:06, 40.4MB/s] 23% 76.5M/332M [00:02<00:06, 40.5MB/s] 24% 80.5M/332M [00:02<00:06, 40.3MB/s] 25% 84.5M/332M [00:02<00:06, 40.8MB/s] 27% 88.5M/332M [00:02<00:06, 41.3MB/s] 28% 92.6M/332M [00:02<00:06, 41.6MB/s] 29% 96.6M/332M [00:02<00:05, 41.7MB/s] 30% 101M/332M [00:02<00:05, 41.8MB/s]  31% 105M/332M [00:02<00:05, 42.0MB/s] 33% 109M/332M [00:02<00:05, 42.0MB/s] 34% 113M/332M [00:02<00:05, 42.4MB/s] 35% 117M/332M [00:03<00:05, 42.6MB/s] 36% 121M/332M [00:03<00:05, 43.1MB/s] 38% 125M/332M [00:03<00:04, 43.6MB/s] 39% 130M/332M [00:03<00:04, 44.6MB/s] 41% 135M/332M [00:03<00:04, 46.1MB/s] 42% 140M/332M [00:03<00:04, 47.8MB/s] 43% 145M/332M [00:03<00:04, 48.8MB/s] 45% 150M/332M [00:03<00:03, 49.9MB/s] 46% 155M/332M [00:03<00:03, 50.5MB/s] 48% 160M/332M [00:03<00:03, 51.1MB/s] 50% 165M/332M [00:04<00:03, 51.5MB/s] 51% 170M/332M [00:04<00:03, 51.8MB/s] 53% 175M/332M [00:04<00:03, 52.1MB/s] 54% 180M/332M [00:04<00:03, 52.3MB/s] 56% 185M/332M [00:04<00:02, 52.5MB/s] 57% 190M/332M [00:04<00:02, 52.7MB/s] 59% 195M/332M [00:04<00:02, 52.7MB/s] 60% 200M/332M [00:04<00:02, 52.7MB/s] 62% 205M/332M [00:04<00:02, 52.9MB/s] 63% 210M/332M [00:04<00:02, 53.1MB/s] 65% 215M/332M [00:05<00:02, 53.2MB/s] 66% 220M/332M [00:05<00:02, 53.3MB/s] 68% 225M/332M [00:05<00:02, 53.5MB/s] 69% 231M/332M [00:05<00:01, 53.4MB/s] 71% 236M/332M [00:05<00:01, 53.6MB/s] 72% 241M/332M [00:05<00:01, 53.6MB/s] 74% 246M/332M [00:05<00:01, 53.8MB/s] 76% 251M/332M [00:05<00:01, 53.6MB/s] 77% 256M/332M [00:05<00:01, 53.9MB/s] 79% 262M/332M [00:05<00:01, 53.9MB/s] 80% 267M/332M [00:06<00:01, 54.0MB/s] 82% 272M/332M [00:06<00:01, 54.0MB/s] 83% 277M/332M [00:06<00:01, 54.1MB/s] 85% 282M/332M [00:06<00:00, 54.1MB/s] 86% 287M/332M [00:06<00:00, 54.2MB/s] 88% 293M/332M [00:06<00:00, 54.2MB/s] 90% 298M/332M [00:06<00:00, 54.3MB/s] 91% 303M/332M [00:06<00:00, 54.0MB/s] 93% 308M/332M [00:06<00:00, 54.0MB/s] 94% 313M/332M [00:06<00:00, 54.0MB/s] 96% 319M/332M [00:07<00:00, 54.2MB/s] 97% 324M/332M [00:07<00:00, 54.2MB/s] 99% 329M/332M [00:07<00:00, 54.4MB/s]100% 332M/332M [00:07<00:00, 47.4MB/s]
[2021-12-21 22:29:34,419] {subprocess.py:89} INFO - Testing 0 1
[2021-12-21 22:29:34,530] {subprocess.py:89} INFO - /opt/conda/lib/python3.8/site-packages/torch/nn/functional.py:3103: UserWarning: The default behavior for interpolate/upsample with float scale_factor changed in 1.6.0 to align with other frameworks/libraries, and now uses scale_factor directly, instead of relying on the computed output size. If you wish to restore the old behavior, please set recompute_scale_factor=True. See the documentation of nn.Upsample for details.
[2021-12-21 22:29:34,530] {subprocess.py:89} INFO -   warnings.warn("The default behavior for interpolate/upsample with float scale_factor changed "
[2021-12-21 22:29:38,695] {subprocess.py:89} INFO - Testing 1 4
[2021-12-21 22:29:43,067] {subprocess.py:89} INFO - Testing 2 5
[2021-12-21 22:29:47,991] {subprocess.py:89} INFO - Testing 3 6
[2021-12-21 22:29:52,986] {subprocess.py:93} INFO - Command exited with return code 0
[2021-12-21 22:29:53,000] {taskinstance.py:1270} INFO - Marking task as SUCCESS. dag_id=data_pipeline, task_id=super_resolution, execution_date=20211219T000600, start_date=20211221T132920, end_date=20211221T132953
[2021-12-21 22:29:53,029] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-12-21 22:29:53,060] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
